### PR TITLE
Added Memories(Photo Gallery) section

### PR DESCRIPTION
**Pull request by Tanya-Goel**
@ashutosh1919 
**With reference to issue no #67** ,
I have added the code for memories section,Please have a look on it!

### CODE REVIEW

**Comment by ashutosh1919** [comment]
@Tanya-Goel , Please resolve conflicts. Refer this: https://stackoverflow.com/a/12093311/8289194

**Comment by Tanya-Goel** [comment]
> @Tanya-Goel , Please resolve conflicts. Refer this: https://stackoverflow.com/a/12093311/8289194

@ashutosh1919 ,I have resolved the conflicts but checks are getting failed.

Is it due to the pictures I uploaded?

File: src/pages/memories/MemoriesComponent.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class Memories extends Component {
+  render() {
+    const theme = this.props.theme;
+    return (
+      <div>
+        <Header theme={theme} />
+        <Fade bottom duration={2000} distance="20px">
+          <div className="heading-div">
+            <h1 className="headingtext" style={{ color: theme.text }}>
+              Some Precious Moments 😇{" "}
```
Comment (ashutosh1919) [inline src/pages/memories/MemoriesComponent.js:19]: For adding emoji you need to take care of some things. 
(1) You need to disable eslint after import like [this](https://github.com/ashutosh1919/masterPortfolio/blob/1f814af67dee3d74bfdba954eda49cd3104c4412/src/components/footer/Footer.js#L5).
(2) You need to wrap emoji into span tag like [this](https://github.com/ashutosh1919/masterPortfolio/blob/1f814af67dee3d74bfdba954eda49cd3104c4412/src/components/footer/Footer.js#L12)

File: src/pages/memories/MemoriesComponent.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import { memories } from "../../portfolio";
+import React, { Component } from "react";
+import { Fade } from "react-reveal";
+import "./MemoriesComponent.css";
+
+const MemoriesData = memories.memoriesSection;
```
Comment (ashutosh1919) [inline src/pages/memories/MemoriesComponent.js:8]: Remove this line. It is not used anywhere.

**Comment by th3c0d3br34ker** [comment]
If this is alright with @Tanya-Goel, @ashutosh1919 give me a day I'll help her out and fix all the issues.

File: src/pages/memories/MemoriesComponent.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import { memories } from "../../portfolio";
```
Comment (ashutosh1919) [inline src/pages/memories/MemoriesComponent.js:3]:
```suggestion
```

Comment (th3c0d3br34ker) [reply src/pages/memories/MemoriesComponent.js]: It should to be there. Since, all the information is import from `portfolio.js` which is not the case.

**Comment by th3c0d3br34ker** [comment]
Topbutton is also not added.

File: src/pages/memories/MemoriesComponent.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import React, { Component } from "react";
```
Comment (th3c0d3br34ker) [inline src/pages/memories/MemoriesComponent.js:3]: import TopButton from "../../components/topButton/TopButton";

File: src/pages/memories/MemoriesComponent.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+          <div className="imgg-div">
+            <img src={require("../../assests/images/img13.JPG")} alt="" />
+            <img src={require("../../assests/images/img11.jpeg")} alt="" />
+          </div>
+          <div className="heading-img-div">
+            <img src={require("../../assests/images/img12.jpg")} alt="" />
+          </div>
+        </Fade>
+        <Footer theme={this.props.theme} onToggle={this.props.onToggle} />
+      </div>
```
Comment (th3c0d3br34ker) [inline src/pages/memories/MemoriesComponent.js:53]: <TopButton theme={this.props.theme} />

**Comment by Tanya-Goel** [comment]
@ashutosh1919 ,
Please review it now. Changes have been made and all checks are getting passed.

**Comment by Tanya-Goel** [comment]
> @Tanya-Goel ,
> Can you please run `npx prettier --write .` to prettify files automatically and to remove unwanted changes. Please push the code after that.

@ashutosh1919 ,I ran the above command  and pushed the code again.

**Comment by saiteja13427** [comment]
I think a new memories page doesn't align with the goals of this project now.

Thus closing this PR and the corresponding issue.

@Tanya-Goel if you still think we should add this, feel free to raise a new issue and we can discuss it further.

**Pull request closed without merging**